save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -502,6 +489,9 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
                     }
                     val newGitDto = gitDto?.copy(url = urlWithTests, branch = newBranch, hash = newCommit)
                         ?: GitDto(url = urlWithTests, branch = newBranch, hash = newCommit)
+
+                    console.log(newGitDto)
```

Comment:
Maybe add this one too into ForbiddenMethodCall in detekt?

Reply:
Mentioned that in https://github.com/analysis-dev/diktat/issues/954 assigned to @nulls

Reply:
Both methods: `['kotlin.js.console.log']` and `['kotlin.js.console']` do not work